README.md

```diff
@@ -1,17 +1,78 @@
-# Robot Framework Page Objects
+# Robot Framework Page Objects 
+## Installing
 
-This Python package adds support of the [Page Object](http://martinfowler.com/bliki/PageObject.html) pattern to [Robot Framework](http://robotframework.org/) & Robot Framework's [Selenium2Library](https://github.com/rtomac/robotframework-selenium2library).  
+    $ pip install robotframework-pageobjects
+## What it is 
+This Python package adds support for the [Page Object](http://martinfowler.com/bliki/PageObject.html) pattern with [Robot Framework](http://robotframework.org/) and Robot Framework's [Selenium2Library](https://github.com/rtomac/robotframework-selenium2library). Though this
+package is a [Robot library](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#extending-robot-framework), it's usable outside the Robot context and facilitates use of the page object pattern independently of any Python testing framework. This means you can use it to create
+page objects and run tests in other testing frameworks like  
+<a href="http://docs.python.org/2/library/unittest.html"><code>unittest</code></a>, 
+[lettuce](http://lettuce.it/tutorial/simple.html) etc. 
+
+In addition to providing a base `Page` class to build upon, this package provides 
+many other conveniences apart from page object modeling including:
+
+- A hidden, yet accessible Selenium2 `driver` instance, which allows you to focus
+on the *application under test* (AUT) instead of Selenium2 implementation details.
+- Easy parameterization, normalization, and setting of testing variables, like the AUT's host,
+the browser type to test with, [Sauce Labs](https://saucelabs.com/) integration, timeouts 
+for locating injected content after page-load etc. 
+- Convenient helper functions like `find_element`, `find_elements` that take [locators](http://rtomac.github.io/robotframework-selenium2library/doc/Selenium2Library.html#Locating%20elements)
+*or* [WebElements](http://selenium-python.readthedocs.org/en/latest/api.html#module-selenium.webdriver.remote.webelement) as parameters.
+- Built-in and readable *assertion methods* on the page object. For example `location_should_be`, `title_should_be` etc.
+- Much more...
+
+Like we said, the package is very flexible: it doesn't force you to use Robot, nor forces you
+to do heavy page object modeling up front. This is great for convicing your organization to move toward
+BDD and page object development because you can approach those ideals iteratively. Even
+with light modeling, and use of a non-BDD framework, your test suites
+can still benefit from the above listed features. Here's an example of a very minimally
+abstracted page object, where we're using a page object assertion (`title_should_be`) 
+outside of the Robot framework. `mytest.py`  
```

Comment:
Is there a better way to state the title of the example file without it being on the same line as that paragraph?